gix-config/Cargo.toml

```diff
@@ -15,6 +15,7 @@ autotests = false
 [features]
 ## Data structures implement `serde::Serialize` and `serde::Deserialize`.
 serde = ["dep:serde", "bstr/serde", "gix-sec/serde", "gix-ref/serde", "gix-glob/serde", "gix-config-value/serde"]
+arbitrary = ["dep:arbitrary", "gix-sec/arbitrary"]
```

Comment:
I'd love to stop the intrusion of `arbitrary` in its tracks at least as long as we don't have `dev-features` for exactly this purpose.

Is there other ways to deal with this?

Reply:
Sure, arbitrary derives can be a quite viral, even when it's an optional dep, there is a non-zero of a maintenance overhead, especially when using third_party types. When implementing fuzzer's this is generally just a convenience thing, but I can definitely manually handle arbitrary types from each fuzzer, it just requires more boilerplate. I'll see what I can come up with.

Reply:
Done